test/dlc_tests/sanity/test_utility_installation.py

```diff
@@ -49,6 +49,7 @@ def test_utility_packages_using_import(training):
         "tensorflow2": "2.4",
         "tensorflow1": "1.15",
         "huggingface_tensorflow": "2.4",
+        "autogluon": "0.2.1",
```

Comment:
can this be removed since this test isn't running on auto gluon? also @tejaschumbalkar - can you confirm if we need utility packages installed for AG containers?

Reply:
removed